Redo Android KBs (#138)
Ostorlab/KB: MOBILE_CLIENT/ANDROID/_LOW/INTENT_SPOOFING/recommendation.md

> To limit one's exposure to this type of attack, developers should: 

Reviewer: ```suggestion
To limit one's exposure to this type of attack, consider the following recommendations: 
```

- Avoid exporting components unless the component is

> specifically designed to handle requests from untrusted applications. 

Reviewer: Don't break line on full sentences. It breaks in the UI.
